go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -84,7 +84,7 @@ type vstreamer struct {
 	format         mysql.BinlogFormat
 	pos            replication.Position
 	stopPos        string
-	lastCommitted  int64
+	commitParent   int64
```

Comment:
While I prefer the terminology of "commit parent", the reason for calling it "last committed" is because that's the term used by MySQL, and so I did not want to introduce a new term